conf/webknossos.latest.routes

```diff
@@ -83,8 +83,9 @@ PUT           /datastores/:name/datasource
 PUT           /datastores/:name/datasources                                     controllers.WKDataStoreController.updateAll(name: String)
 PATCH         /datastores/:name/status                                          controllers.WKDataStoreController.statusUpdate(name: String)
 POST          /datastores/:name/verifyUpload                                    controllers.WKDataStoreController.validateDataSetUpload(name: String)
-POST          /datastores/:name/addInitialTeams                                 controllers.WKDataStoreController.addDataSetInitialTeams(name: String, token: String, dataSetName: String)
+POST          /datastores/:name/reportDatasetUpload                             controllers.WKDataStoreController.reportDatasetUpload(name: String, token: String, dataSetName: String, dataSetSizeBytes: Long)
 POST          /datastores/:name/deleteErroneous                                 controllers.WKDataStoreController.deleteErroneous(name: String)
+POST          /datastores/:name/reportIsosurfaceRequest                         controllers.WKDataStoreController.reportIsosurfaceRequest(name: String, token: Option[String])
```

Comment:
Should this be tracked from the frontend?

Reply:
Yes I think I’ll move it back to the front-end when implementing #5166